src/components/App.js

```diff
@@ -0,0 +1,70 @@
+import { css, html, LitElement } from "lit";
+import { customElement } from "lit/decorators.js";
+import { ref } from "lit/directives/ref.js";
+import { router } from "/router.js";
+
+class AppElement extends LitElement {
+  static get styles() {
+    return css`
+      h1 {
+        margin: 0;
+      }
+
+      header,
+      main {
+        padding: 1rem;
+      }
+
+      header {
+        background-color: #403539;
+        color: white;
+        box-shadow: 0 0 0 3px hsl(340 20% 6% / 0.95);
+      }
+
+      nav {
+        display: flex;
+        align-items: center;
+      }
+
+      a {
+        color: inherit;
+      }
+
+      a:not(:hover) {
+        text-decoration: none;
+      }
+
+      ul {
+        margin-left: auto;
+        list-style: none;
+        padding: 0;
+        display: flex;
+      }
+    `;
+  }
+
+  #updateOutlet(node) {
```

Comment:
why are you adding `#` as a prefix? is it part of Lit? Do you have the documentation link for this?

Reply:
@JuanVqz It's JavaScript native [Private Class Method](https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Classes/Private_class_fields) syntax

Reply:
Interesante 🧐